apps/sensenet/src/components/dialogs/upload/upload-dialog.tsx

```diff
@@ -225,9 +240,10 @@ export function UploadDialog(props: UploadDialogProps) {
           ev.target.files && addFiles([...ev.target.files])
         }}
         style={{ display: 'none' }}
+        disabled={props.disableMultiUpload && files && files.length > 0}
```

Comment:
props.disableMultiUpload && files?.length

ez is elég

Reply:
Unfortunatelly not enough  😢 
![image](https://user-images.githubusercontent.com/22347393/90601692-e21e5880-e1f8-11ea-8229-d3b54548c873.png)

Reply:
![image](https://user-images.githubusercontent.com/22347393/91014646-0eb3e500-e5ea-11ea-9ce2-0ac7530d9be5.png)